### PR TITLE
Docs/ns frame content

### PR DESCRIPTION
**Pull request by RobTobias123**
Added overview paragraphs and Example placeholder

### CODE REVIEW

**Comment by 7AXEL** [comment]
Check out this project: https://github.com/7AXEL/Web2APK

**Comment by andij** [comment]
I expect to have just one Example file.

- standard.html

The other examples taken from Storybook, are combinations of content and config. Which can be toggled and changed in the Configurator.

With one example, it's not necessary to be in <Tabs> (refer to ns-accordion).

File: src/assets/components/ns-frame/standard.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+          <dd>Venmo trust fund vexillologist church</dd>
+        </div>
+      </dl>
+      <ns-download href="this-text-gets-ignored.pdf" filesize="123MB">This text is also used for the downloaded file name</ns-download>
+      <ns-cta>Paleo fashion</ns-cta>
+    </ns-content>
+    </ns-panel>
+<ns-footer>
+</ns-footer>
+</ns-frame>
```
Comment (andij) [inline src/assets/components/ns-frame/standard.html:31]:
```suggestion
<ns-frame active-button="0" buttons="[{&quot;icon&quot;:&quot;home&quot;,&quot;label&quot;:&quot;Tonx affogato&quot;,&quot;url&quot;:&quot;#!&quot;},{&quot;icon&quot;:&quot;bill&quot;,&quot;label&quot;:&quot;Biodiesel&quot;,&quot;url&quot;:&quot;#!&quot;},{&quot;icon&quot;:&quot;meter&quot;,&quot;label&quot;:&quot;Man bun&quot;,&quot;url&quot;:&quot;#!&quot;},{&quot;icon&quot;:&quot;bar-chart&quot;,&quot;label&quot;:&quot;Tacos&quot;,&quot;url&quot;:&quot;#!&quot;},{&quot;icon&quot;:&quot;tariff&quot;,&quot;label&quot;:&quot;Chillwave&quot;,&quot;url&quot;:&quot;#!&quot;},{&quot;icon&quot;:&quot;question&quot;,&quot;label&quot;:&quot;Squid&quot;,&quot;url&quot;:&quot;#!&quot;}]" menu="[{&quot;label&quot;:&quot;Small batch&quot;,&quot;data&quot;:[{&quot;label&quot;:&quot;Snackwave&quot;,&quot;value&quot;:&quot;Raw denim etsy master cleanse forage&quot;,&quot;links&quot;:[{&quot;label&quot;:&quot;Book glossier&quot;,&quot;url&quot;:&quot;#!&quot;,&quot;icon&quot;:&quot;location-outline&quot;}]},{&quot;label&quot;:&quot;Mixtape fashion&quot;,&quot;value&quot;:&quot;Poutine cornhole celiac&quot;,&quot;links&quot;:[{&quot;label&quot;:&quot;Cleanse forage&quot;,&quot;url&quot;:&quot;#!&quot;,&quot;icon&quot;:&quot;settings&quot;},{&quot;label&quot;:&quot;Raw denim&quot;,&quot;url&quot;:&quot;#!&quot;,&quot;icon&quot;:&quot;home-outline&quot;}]},{&quot;label&quot;:&quot;Neutra flannel bag&quot;,&quot;links&quot;:[{&quot;label&quot;:&quot;Distillery freegan&quot;,&quot;url&quot;:&quot;#!&quot;,&quot;icon&quot;:&quot;protect-outline&quot;}]}]}]">
  <main id="content">
    <ns-landmark type="lakeside">
      <h1 slot="heading">Live-edge meggings</h1>
      <p slot="paragraph">activated charcoal sartorial pug same lomo grailed cornhole</p>
    </ns-landmark>
    <ns-panel>
      <ns-column hockeypitch="3">
        <ns-card type="section" decoration="green-leaf">
          <h3 slot="heading">Ethical neutra hexagon</h3>
          <div slot="paragraph">
            <p class="h4">06794</p>
            <p>Helvetica hexagon: <b><ns-date-time date="2024-06-17"></ns-date-time></b></p>
          </div>
          <ns-cta slot="cta" type="text">Submit a reading</ns-cta>
        </ns-card>
        <ns-card type="section" decoration="protect">
          <h3 slot="heading">Bespoke bitters edison bulb</h3>
          <div slot="paragraph">
            <p class="h4">34767</p>
            <p>Helvetica hexagon: <b><ns-date-time date="2024-06-17"></ns-date-time></b></p>
          </div>
          <ns-cta slot="cta" type="text">Submit a reading</ns-cta>
        </ns-card>
        <ns-card type="section" decoration="payments">
          <h3 slot="heading">Helvetica kombucha</h3>
          <div slot="paragraph">
            <p>Solarpunk bitters health goth, authentic jianbing marfa yuccie intelligentsia.</p>
          </div>
          <div slot="cta">
            <ns-cta type="text">Switch tariff</ns-cta>
          </div>
          <div slot="cta">
            <ns-cta type="text">Tariff details</ns-cta>
          </div>
        </ns-card>
      </ns-column>
    </ns-panel>
  </main>
  <ns-footer id="footer"></ns-footer>
</ns-frame>
```

File: src/content/docs/components/ns-frame.mdx [hunk 2]
```diff
@@ -15,8 +15,16 @@ import { Tabs, TabItem } from '@astrojs/starlight/components';
...
 
+The `ns-frame` component comprises a header and buttons that are adjacent to an anonymous content area. When a button is clicked, it navigates through the content. This component should not be used in conjunction with the `ns-header`. It is particularly useful in scenarios such as when a user is logged into their account.
+
 ### Examples
 
+<Tabs>
+  <TabItem label="Standard">
+    <ExampleViewer type="standard" />
+  </TabItem>
+</Tabs>
```
Comment (andij) [inline src/content/docs/components/ns-frame.mdx:26]: When there is only one example, don't include the `<Tabs>` and `<TabItem>`.
```suggestion
<ExampleViewer type="standard" />
```

**Comment by RobTobias123** [comment]
Resolved conflict on badges.json

File: src/data/badge.json [hunk 3]
```diff
@@ -1,9 +1,5 @@
 {
   "components": [
-    {
-      "name": "ns-landmark",
-      "badge": "updated"
-    },
```
Comment (andij) [inline src/data/badge.json:6]: The `updated` badge is still required for landmark.

**Pull request merged**